pkg/controller/operators/catalog/operator.go

```diff
@@ -1627,6 +1637,14 @@ func (o *Operator) syncInstallPlans(obj interface{}) (syncError error) {
 	if len(plan.Status.BundleLookups) > 0 {
 		unpacked, out, err := o.unpackBundles(plan)
 		if err != nil {
+			// If the error was fatal capture and fail
+			if fatal := olmerrors.IsFatal(err); !fatal {
```

Comment:
```suggestion
			if fatal := olmerrors.IsFatal(err); fatal {
```
Since in this case, if the error is fatal, we want to transition the installplan to failed